pages/common/gh-release.md

```diff
@@ -9,20 +9,20 @@
 
 - Display information about a specific release:
 
-`gh release view {{tag}}`
+`gh release view "{{tag}}"`
```

Comment:
This syntax suggests that double quotes around the tag name are required, which doesn't appear to be the case?

Reply:
I don't know whether tags can contain spaces that's why I put quotes here.